packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -152,13 +166,42 @@ impl Abigen {
                 use fuels::core::{Detokenize, EnumSelector, ParamType, Tokenizable, Token};
 
                 #code
+                #deployable_trait
 
                 #abi_structs
                 #abi_enums
             }
         })
     }
 
+    fn deployable_trait(&self, name: &Ident) -> TokenStream {
```

Comment:
The function should be name `generate_deployable_trait` or similar. I'm not sure of the relevance of it though.